cmake/OpenSimMacros.cmake

```diff
@@ -143,3 +143,21 @@ FUNCTION(OPENSIM_ADD_LIBRARY)
     ENDIF()
 
 ENDFUNCTION()
+
+# Copy a file from the directory containing test resources (model files, data,
+# etc.) to the directory in which a test will be executed. This function makes
+# it easy to re-use files that are used in tests. With an easier mechanism for
+# re-using resources, we won't end up version-controllingthe same file into
+# multiple test directories.
+#
+# Arguments are a list of files in the test resources directory
+# (OPENSIM_TEST_RESOURCES_DIR) to copy.
+FUNCTION(OPENSIM_COPY_TEST_RESOURCES)
```

Comment:
What do you think about `opensim_copy_test_resources`? I think function names are case insensitive. I still plan to descreamify the entire repo; is it okay if I keep screamy consistency across the repo for now, and descreamify separately?

However, using camelcase also helps distinguish real cmake methods from ones we've made (though I've tried to do this by putting OpenSim in the name of the function).